python/paddle/fluid/dygraph/dygraph_to_static/static_analysis.py

```diff
@@ -260,20 +260,25 @@ def get_node_to_wrapper_map(self):
     def get_var_env(self):
         return self.var_env
 
+    def _get_constant_node_type(self, node):
+        assert isinstance(node, gast.Constant), \
+            "Type of input node should be gast.Constant, but received %s" % type(node)
+        # singleton: None, True or False
+        if node.value is None:
+            return {NodeVarType.NONE}
+        if isinstance(node.value, bool):
+            return {NodeVarType.BOOLEAN}
+        if isinstance(node.value, int):
+            return {NodeVarType.INT}
+        if isinstance(node.value, float):
+            return {NodeVarType.FLOAT}
+        if isinstance(node.value, str):
+            return {NodeVarType.STRING}
```

Comment:
Could you return {NodeVarType.UNKNOWN} at last of _get_constant_node_type in case that these is a case we didn't consider? Other things LGTM

Reply:
You're right. I already add commit to fix it.